src/flux_workarounds.jl

```diff
@@ -0,0 +1,57 @@
+using Flux: BatchNorm, InstanceNorm, GroupNorm, Params, trainable
+using Base: IdSet
+export weights, loadweights!
+
+"""
+    LegolasFlux.other_weights(layer) -> Vararg{Array}
+
+Given a layer with params that are not captured by `Flux.trainable`, produce
+a tuple of arrays corresponding to these parameters (analogous to `Flux.trainable`).
+"""
+function other_weights end
```

Comment:
what does this line do, given the default implementation below?

Reply:
I did this so the docstring would be attached to the function as a whole, instead of a particular method. It doesn't matter here since none of the others have docstrings anyway, but we could add a separate docstring for each method and if you do `? other_weights` or `? other_weights(layer)` it will show you all the docstrings for all the methods that fit that signature, so we could just put it on the fallback method and it would always show up, just like it will when we attach it to the function itself like I did here (see https://docs.julialang.org/en/v1/manual/documentation/#Functions-and-Methods for probably a better explanation). But really I just did it because I felt like conceptually that docstring belongs to the whole function and I felt like it helped me formulate the API. Like here's the function, here's what it does in words, and then here are the methods to do that.

Reply:
oh cool! thanks for explaining. :)